package.json

```diff
@@ -11,10 +11,12 @@
     "bootstrap": "^4.3.1",
     "dotenv": "^8.0.0",
     "mapbox-gl": "^0.52.0-beta.2",
+    "mapbox-gl-arcgis-tiled-map-service": "^0.3.0",
```

Comment:
This dependency can be removed from master at some point since it is no longer be used for the aerial layers.

Reply:
good catch